File: 1070.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var b = Number(lines.shift());
var vezes = 0;
while(vezes !== 6){
    if(b % 2 !== 0){
        console.log(b)
        var vezes = vezes + 1}
    var b = b + 1}

File: 1070.java
import java.util.Scanner;
public class Main {
public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int b = input.nextInt(), vezes = 0;
        while(vezes != 6) {
                if(b % 2 != 0) {
                        System.out.println(b);
                        vezes += 1;
                }
                b += 1;
        }
}
}
